geomesa-process/geomesa-process-vector/src/main/scala/org/locationtech/geomesa/process/transform/ArrowConversionProcess.scala

```diff
@@ -67,7 +68,9 @@ class ArrowConversionProcess extends GeoMesaProcess with LazyLogging {
               @DescribeParameter(name = "batchSize", description = "Number of features to include in each record batch", min = 0)
               batchSize: java.lang.Integer,
               @DescribeParameter(name = "flattenStruct", description = "Removes the outer SFT struct yielding top level feature access", min = 0)
-              flattenStruct: java.lang.Boolean
+              flattenStruct: java.lang.Boolean,
+              @DescribeParameter(name = "axisOrder", description = "Geometry axis order: LatLon or LonLat", min = 0, defaultValue = "LatLon")
+              axisOrder: AxisOrder = LatLon
```

Comment:
```suggestion
              @DescribeParameter(name = "flipAxisOrder", description = "Flip the axis order of returned coordinates from latitude/longitude (default) to longitude/latitude", min = 0)
              flipAxisOrder: java.lang.Boolean
```

Reply:
- So I went with a hybrid approach: making the public API be very explicit, forcing the user to specify an enum `LatLon` or `LonLat` which internal is then translated and treated like a Boolean. The main reasons for doing so were that it communicates the intent clearly and the concern you had about adding additional dependencies for the enum is no longer there since the `utils` package is already being brought in and referenced virtually everywhere.

- I would suggest we leave this as is having the user specify the axis order and making it have a default value, and the internal logic figure out if it needs to flip or not, wdyt?

Reply:
one concern I have here is that these variables get serialized from xml when called through geoserver, e.g. things like [this](https://github.com/locationtech/geomesa/blob/main/docs/user/_static/process/UniqueProcess_wps.xml). I'm not sure how serialization will work for a custom axis order enum. If you want to keep this, could you test that out? But also, I think it's better to favor consistency, i.e. have the hints that trigger an analytic query correspond to the wps parameters that do the same thing. And as you say, we'd need internal logic to calculate the correct axis order, and I don't have a good sense of how to do that correctly, given all the complexities described in the [geoserver docs](https://docs.geoserver.org/main/en/user/services/wfs/axis_order.html#axis-ordering), so I'd rather punt on it.

Reply:
I'm running this of a Mac and can't run the tests will switch it over to a Boolean